plugins/@grouparoo/hubspot/src/lib/export-contacts/connection.ts

```diff
@@ -20,7 +20,14 @@ export const contactsDestinationConnection: PluginConnection = {
   apps: ["hubspot"],
   syncModes: contactsSupportedSyncModes,
   defaultSyncMode: "sync",
-  options: [],
+  options: [
+    {
+      key: "companyKey",
+      displayName: "Company Key",
+      required: false,
+      description: "Which field in the Company uniquely represents it?",
```

Comment:
If companyKey is optional, what will be used by default?  We might want to mention the email domain will be used if this is not set.

Reply:
But doesn't hubspot make companies autoamtically from the email domain?  Either way, 👍

Reply:
This could say something like "Links a Contact to a company by something other than the Hubspot default (email domain)."

Reply:
ahh! Now I got what you're saying @evantahler, thank you. Brian's text will do the trick.